moto/core/models.py

```diff
@@ -788,6 +789,27 @@ def __call__(self, func=None):
         else:
             return mocked_backend
 
+ecs_backends_global = {}
+class atidot_base_decorator(base_decorator):
+    mock_backend = MockAWS
+
+    def __init__(self, backends):
+        super().__init__(backends)
+
+    def __call__(self, func=None, callback=None):
```

Comment:
I don't think we need this implementation at all

Reply:
@barakber  that's the place which we get the callback from decorator